deploy-config/deploy-config-holesky.toml

```diff
@@ -5,16 +5,17 @@ chain_id = 17000
 # ======================
 
 [dual_governance]
-admin_proposer = "0xdA7d2573Df555002503F29aA4003e398d28cc00f"                 # DAO Voting
-proposals_canceller = "0xdA7d2573Df555002503F29aA4003e398d28cc00f"            # DAO Voting
-reseal_committee = "0x17A283c01dCc0F1d3D67b24E341e142DF78807D4"               # Gnosis Multisig
-sealable_withdrawal_blockers = ["0xC455F41f0A5a01359B6044b762A85B90324Ea5F0"] # Mock Withdrawal queue
+
+admin_proposer = "0xdA7d2573Df555002503F29aA4003e398d28cc00f"
+proposals_canceller = "0xdA7d2573Df555002503F29aA4003e398d28cc00f"
+sealable_withdrawal_blockers = ["0xc7cc160b58F8Bb0baC94b80847E2CF2800565C50", "0xffDDF7025410412deaa05E3E1cE68FE53208afcb"] 
+reseal_committee = "0x891BCE021d39557E81EA465989869581829C266f"               # Gnosis Multisig TBD
 tiebreaker_activation_timeout = 900
```

Comment:
Let's add comments to the durations and timestamps, describing their values. For example:
```toml
tiebreaker_activation_timeout = 900 # 15 min
```